Pipfile.lock

```diff
@@ -1,11 +1,11 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "a591778cf707b35ec7cfff3e47e24c8f630c14398e8aa78fec4225dd21c8a2f4"
+            "sha256": "80676ff8d37ab0d57717a6ddcf9f5f601e2071b3eadc0a6a348110531c7de91e"
```

Comment:
The changes in this file aren't related to this change in any way I don't think? 

I guess there's no harm leaving it in this PR this though.

Reply:
Yep, they're not. I've just included it as part of this PR since anyone doing a `pipenv install` would include this file as part of their PR